Gemfile

```diff
@@ -58,8 +58,13 @@ group :development do
 end
 
 group :test do
+  gem "rails-controller-testing"
```

Comment:
Hmm, do we want it? What does it give us exactly?

Reply:
Oh, I just ran into it on a different PR. Lets us check what was rendered. 

Let's keep it :)

Reply:
Everywhere you see `assigns` or `render_template` in #5, that's this gem

Reply:
Perhaps we want to test in a different way though